src/main/cpp/src/parse_uri.cu

```diff
@@ -845,7 +917,8 @@ std::unique_ptr<column> parse_uri(strings_column_view const& input,
     input.chars_begin(stream),
     offsets_mutable_view.begin<size_type>(),
     reinterpret_cast<size_type*>(src_offsets.data()),
-    reinterpret_cast<bitmask_type*>(null_mask.data()));
+    reinterpret_cast<bitmask_type*>(null_mask.data()),
```

Comment:
Not specific to this PR, but do we need to create an all-valid bitmask here if the input is not nullable?  Why not just leave the output not-nullable as well?